assembl/tests/views_tests/test_graphql.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+import json
+
+import pytest
+from graphql_relay.node.node import to_global_id
+
+from assembl import models
+from assembl.graphql.schema import Schema as schema
+
+
+@pytest.fixture(scope="function")
```

Comment:
This fixture makes sense only in this module. It is not meant to be shared in other modules. I'd prefer it to be colocated with the tests. If later we have several test modules where it can be useful, yeah it makes sense to move it now. I'll do it.